pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -113,13 +169,15 @@ stdenv.mkDerivation rec {
   ];
 
   nativeBuildInputs = [
+    gnused
     zip
     python
     unzip
     makeWrapper
     which
     customBash
-  ] ++ lib.optionals (stdenv.isDarwin) [ libcxx CoreFoundation CoreServices Foundation ];
+    binutils
+  ] ++ lib.optionals (stdenv.isDarwin) [ cctools clang Libsystem libcxx CoreFoundation CoreServices Foundation ];
```

Comment:
https://github.com/NixOS/nixpkgs/pull/42832/files#r199536836

> Clang shouldn't be needed - it will be provided by stdenv.

Is there no way of having a stdenv with gcc instead of clang on darwin? If so, should I replace all instances of `${clang}` with `${stdenv.cc}`?

Reply:
yes, `gccStdenv` or `clangStdenv` for the reverse on linux.